### PR TITLE
[DOCS] Glossary Update

### PR DESCRIPTION
**Pull request by kwcanuck**
Removed unnecessary images and headings from Glossary topics. Further clean up is required.

### Definition of Done
- [X] I have made corresponding changes to the documentation
- [X] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: docs/docusaurus/docs/terms/datasource.md [hunk 1]
```diff
@@ -3,63 +3,42 @@ title: Datasource
...
 hoverText: Provides a standard API for accessing and interacting with data from a wide variety of source systems.
 ---
-
-import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
-import ConnectHeader from '/docs/images/universal_map/_um_connect_header.mdx';
-import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
-import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
 import TechnicalTag from '../term_tags/_tag.mdx';
 
 import InProgress from '/docs/components/warnings/_in_progress.md'
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/terms/datasource.md]: I think we may be able to remove this now.

**Review by Rachel-Reverie — APPROVED** [review]
LGTM.  I had one specific suggestion on the Datasource file.  I also think that pulling the "features" sub headers from H3 up to H2 level makes them a seem a bit random when they are included (I do understand that having them under an otherwise empty H2 "Features" header is poor formatting, I just feel that they work better when grouped and provided that context), but I'll bow to your thoughts there.

**Comment by kwcanuck** [comment]
> LGTM. I had one specific suggestion on the Datasource file. I also think that pulling the "features" sub headers from H3 up to H2 level makes them a seem a bit random when they are included (I do understand that having them under an otherwise empty H2 "Features" header is poor formatting, I just feel that they work better when grouped and provided that context), but I'll bow to your thoughts there.

@Rachel-Reverie you raise valid points. However, these Glossary entries are far too long and need to be further reduced. Ideally, we should only be providing a definition and not benefits, features, or configuration information. As you indicated, this content will likely be removed or relocated in the near future. I think it's best to make an iterative change now, and revisit the content and structure later as time permits. I agree that the heading structure isn't ideal, but I think it's easier to navigate than it was previously.

File: docs/docusaurus/docs/terms/datasource.md (hunk 1 above)
Comment (kwcanuck) [reply docs/docusaurus/docs/terms/datasource.md]: @Rachel-Reverie - done!

**Pull request merged**